core/src/main/java/org/infinispan/xsite/XSiteAdminCommand.java

```diff
@@ -104,8 +104,8 @@ public void writeTo(ObjectOutput output) throws IOException {
             return;
          case AMEND_TAKE_OFFLINE:
             output.writeUTF(siteName);
-            output.writeInt(afterFailures);
-            output.writeLong(minTimeToWait);
+            output.writeObject(afterFailures);
+            output.writeObject(minTimeToWait);
```

Comment:
@pruivo because there are very few `XSiteAdminCommand` instances ever serialized, so the memory savings are negligible? The risk of someone requiring an existing field for a particular command type and forgetting to modify the switch is non-zero, and I don't think the memory savings are worth the pain to debug such a scenario.